source/panel_predictor/panel_finder/panel_classifier/__main__.py

```diff
@@ -1,4 +1,4 @@
-from source.panel_finder.panel_classifier.panel_classifier import PanelClassifier
+from source.panel_predictor.panel_finder.panel_classifier.panel_classifier import PanelClassifier
```

Comment:
Does `from .panel_classifier import PanelClassifier` work?

Reply:
Apparently it does. This could be changed on every file that imports a module. Do you think it should be?

Reply:
actually no, this does not work.